client/plugins/i18n.js

```diff
@@ -30,9 +31,10 @@ export default ({ app, store, req }) => {
 
   // Set i18n instance on app
   // This way we can use it in middleware and pages asyncData/fetch
+  const sessionLang = store.getters['session/lang']
   app.i18n = new VueI18n({
-    locale: store.state.locale || 'zh',
-    fallbackLocale: 'zh',
+    locale: sessionLang,
+    fallbackLocale: fallbackLang,
     messages: { en, fr, zh }
```

Comment:
This will be reworked so that we use a Side-Car `/hpi/ui/localization` (TBD) from where we could ask another system to give appropriate messages object.

Idea would be that default build would have minimum entries (for smaller as possible package), and another system would provide translation. Besides, changing translations should not require a build.

What has to be figured out is how to merge received data (e.g. **nuxtjs/i18n** [syncVuex](https://github.com/nuxt-community/nuxt-i18n/blob/master/src/helpers/utils.js#L127), and/or [setMessages](https://github.com/nuxt-community/nuxt-i18n/blob/master/src/plugins/main.js#L34), and/or [loadLanguagesAsync](https://github.com/nuxt-community/nuxt-i18n/blob/master/src/plugins/main.js#L75)).

TBD.